lib/ReactCrop.js

```diff
@@ -279,7 +279,9 @@ class ReactCrop extends PureComponent {
       onChange,
     } = this.props;
 
-    if (e.target !== this.imageRef) {
+    const componentEl = this.mediaWrapperRef.firstChild;
+
+    if (e.target !== componentEl || !componentEl.contains(e.target)) {
```

Comment:
I think this will only work if a `<video>` is passed in directly. It might be nice to allow the user to pass in a `<div>` containing the video instead so they can set the size they want it to display.

Reply:
Could do something like `mediaWrapperRef.contains(e.target))` instead, I think that would solve the problem

Reply:
Yes good catch you might need to refresh as I did exactly that 😉

Reply:
Yes good catch you might need to refresh as I did exactly that 😉

Reply:
Perfect! Thanks for taking the time to do this 😊

Reply:
Np it's slightly less convenient for you in the situation that you want to specify a percentage crop with a fixed aspect (and omit width or height), as `onImageLoaded` would complete the width or height of such a crop. It will still be possible to do something yourself in onLoadStart (if I recall the video load event name right) but you'd have to calculate the aspect crop yourself based off the video dimensions